app/templates/events/event_list.html

```diff
@@ -96,12 +96,20 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
               {% set link_class = 'link-secondary' if event.isPast else 'link-primary' %}
                 <td><a href="/event/{{event.id}}/edit" class="{{link_class}}">{{event.name}}</a></td>
             {% else %}
-              {% set link_class = 'link-secondary' if event.isPast else 'link-primary' %}
-              <td><a href="/event/{{event.id}}/view" class="{{link_class}}">{{event.name}}</a></td>
+              <td><a href="/event/{{event.id}}/view" class="link-primary">{{event.name}}</a></td>
             {% endif %}
-            <td nowrap>{{event.startDate.strftime('%m/%d/%Y')}}</td>
-            <td nowrap>{{event.timeStart.strftime('%-I:%M %p')}} - {{event.timeEnd.strftime('%-I:%M %p')}}</td>
-            <td class="text-dark">{{event.location}}</td>
+              {% if event.isCanceled %}
+                {% if managersProgramDict[event.program] and managersProgramDict[event.program]["managers"] | length > 2 %} 
+                  <td colspan = 3 >This event has been canceled. Contact <a href="mailto:{{event.contactEmail}}" class="text-dark">{{managersProgramDict[event.program]["managers"]}}</a> if you have any questions.</td>
+                {% else %}
+                  <td colspan = 3 >This event has been canceled. Contact <a href="mailto:celts@berea.edu;" class="text-dark">CELTS</a> if you have any questions.</td>
+                {% endif %}
+              {% else %}
+              {% set id = event.id %}
```

Comment:
Why are you doing this set? You never use it and it would be better to just use event.id if you needed to use it since it would be more readable.